Dockerfile

```diff
@@ -23,6 +23,7 @@ RUN apt-get update \
 
 ENV PYTHON=python3
 COPY package.json yarn.lock /usr/src/app/
+RUN npm install typescript -g
```

Comment:
why do we need to install typescript ?
should it not be already installed by `yarn install` if needed ?

Reply:
When building the sub dependencies, especially scubaclient, we have an error where tsc is not found (for scubaclient), because we need to install tsc globally first. I did not try to dig into it much, but it's pretty common to install tsc globally in dockerfiles of ts projects (or projects using it).